core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -43,13 +43,15 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
       <div class="title-section-mask"></div>
 
       <div ng-attr-section="<[isWindowLarge ? undefined : 'right-section']>">
-        <div ng-if="isWindowLarge" class="objective protractor-test-exp-summary-tile-objective">
-          <span ng-if="getObjective()">
-            <[getObjective() | truncateAndCapitalize: 40]>
-          </span>
-          <span ng-if="!getObjective()" translate="I18N_LIBRARY_NO_OBJECTIVE">
+        <span ng-if="getObjective()">
+          <span tooltip="<[getObjective()]>" tooltip-placement="top">
+            <div ng-if="isWindowLarge" class="objective protractor-test-exp-summary-tile-objective">
```

Comment:
Having a div inside a span is a bit weird. Will this mess up the UI if it's a span instead?